pkg/api/handlers/libpod/manifests.go

```diff
@@ -90,6 +90,16 @@ func ManifestInspect(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if len(schema2List.Manifests) == 0 {
+		var schema2 manifest.Schema2
+		if err := json.Unmarshal(rawManifest, &schema2); err != nil {
+			utils.Error(w, "Something went wrong.", http.StatusInternalServerError, err)
+			return
+		}
+		utils.WriteResponse(w, http.StatusOK, schema2)
```

Comment:
@restanrm Should we add a check here and return empty list if`schema2` is actually empty or no-manifest is present instead of "Something went wrong" . I think it is more RESTful to return `empty list` instead of "Something went wrong". @jwhonce Could confirm.

Right question would be what happens when `len(schema2List.Manifests) == 0` and `schema2` is also empty.

Reply:
@flouthoc I've updated the PR to check for `len(schema2.LayersDescriptors) == 0 `. This should return a 204 error with empty body. The `Something went wrong` message is returned when unmarshalling json failed, it should be a quite serious error in my opinion.